mac_launch_docker.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
+#This file launches spinmob in a display-ready docker container
+
+#Credit to: http://stackoverflow.com/a/36190462/2689923
+
+#Requires Xquartz and socat installed!
+
+#Define what we are running here.
+CONTAINER=spinmob
+COMMAND=/bin/bash
+NIC=en0
+
+# Grab the ip address of this box
```

Comment:
# comment looks better than 
#comment

I don't know what the bash standard is.